RELEASE_NOTES.md

```diff
@@ -12,8 +12,76 @@ openssl<sup>1</sup> | >=v1.x
 <sup>1</sup> openssl is needed if libdrizzle-redux is compiled with support for
 SSL connections.
 
+New project layout
+==================
+
+A more general project layout has been adopted such that source files are now
+located in the folder `src` and headers for the public interface are in `include/libdrizzle-redux`.
+
+Migration Instructions
+======================
+
+- header file include path
+
+    Projects using **libdrizzle-redux** header files should change `#include`
+    directives from `#include <libdrizzle-5.1/[HEADERFILE].h>` to
+    `#include <libdrizzle-redux/[HEADERFILE].h>`.
+
+- `binlog.h`
+
+    The signature of the function `drizzle_binlog_get_filename` changed from
+
+    `void drizzle_binlog_get_filename(drizzle_st* con, char** filename, int file_index)`
+
+    to
+
+    `drizzle_return_t drizzle_binlog_get_filename(drizzle_st* con, char** filename,
+    uint32_t* end_position, int file_index)`
+
+- `constants.h`
+
+    The definition of the callback function for log handling `drizzle_log_fn`
+    has changed from
+
+    `typedef void (drizzle_log_fn)(const char *line, drizzle_verbose_t verbose,
+    void *context);`
+
+    to
+
+    `typedef void (drizzle_log_fn)(const char *file, uint line, const char *func,
+    const char *msg, drizzle_verbose_t verbose, void *context)`
```

Comment:
Occurs to me that this could have been made a `struct` for better forward compatibility. Next time ?